bin/prompt_proto_upload_raws.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# This file is part of prompt_prototype.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# This script uploads the raw files from the ap_verify_ci_cosmos_pdr2 repo
+# to Google Storage. It renames the files to match prompt_prototype conventions.
+
+set -e  # Abort on any error
+
+RAW_DIR="${AP_VERIFY_CI_COSMOS_PDR2_DIR:?'dataset is not set up'}/raw"
+
+gsutil cp "${RAW_DIR}/HSC-0059150-050.fits.gz" \
+    gs://rubin-prompt-proto-unobserved/HSC/50/2016030700001/0/HSC-2016030700001-0-0059150-HSC-G-050.fits.gz
```

Comment:
Maybe a comment above these that notes that the directory structure is built to mimic that of the butler.

Reply:
It doesn't mimic the Butler (in fact, I think K-T mentioned that it's deliberately different from the Butler). It mimics what's in `upload.py`/`activator.py`.